addons/website/models/ir_ui_view.py

```diff
@@ -3,14 +3,44 @@
 import logging
 
 from lxml import etree, html
+from HTMLParser import HTMLParser
 
 from openerp import SUPERUSER_ID, api, tools
 from openerp.addons.website.models import website
 from openerp.http import request
 from openerp.osv import osv, fields
+from openerp.tools.safe_eval import safe_eval
 
 _logger = logging.getLogger(__name__)
 
+class AssetsExtractor(HTMLParser):
+    js=[]
+    css=[]
+    less = []
```

Comment:
Not need to defined here: js, css and less (because all values are defined in __init__